packages/clerk-js/src/core/state.ts

```diff
@@ -36,6 +48,14 @@ export class State implements StateInterface {
     eventBus.on('resource:fetch', this.onResourceFetch);
   }
 
+  get __internal_waitlist() {
+    if (!this._waitlistInstance) {
+      this._waitlistInstance = new Waitlist(null);
+      this.waitlistResourceSignal({ resource: this._waitlistInstance });
+    }
+    return this._waitlistInstance;
+  }
```

Comment:
The proxy layer needed a way to access a waitlist instance, and so this property was added. It would be nice if there was a way to avoid having to do this though... 🤔